helpers.c

```diff
@@ -31,7 +31,7 @@ char* formatUserStatus(int status)
 
 char* formatTransactionStatus(int status)
 {
-    char* trans_s[] = {"none", "open", "close", "past due" };
+    char* trans_s[] = {"none", "open", "close", "past due", "purchased" };
```

Comment:
Add a type field in transaction class, when we start this project in c++. For now we can skip it